inst/rmarkdown/templates/severity/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,326 @@
+---
+title: "Estimating severity and underascertainment"
+author: Carmen Tamayo, Hugo Gruson
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_document:
+    code_folding: "hide"
+link-citations: no
+params:
+  data_file: 
+    label: "Name of file containing cases and deaths over time"
+    value: "data/Marburg_EqGuinea_linelist.csv"
+    input: file
+  data_type:
+    label: "Indicate the type of data to be used in this report"
+    value: "Linelist"
+    choices: ["Linelist", "Aggregated"]
+  disease_name: 
+    label: "Name of the disease of interest, also used to select diseases in the epiparameter database if `use_parameter_database = TRUE`."
+    value: "Ebola"
+  data_columns:
+    label: "Columns present in your dataset"
+    value: "Dates of onset and total deaths only"
+    choices: ["Dates of onset and total deaths only", "Dates of onset and death", "Dates of onset and whether case died"]
+  use_epiparameter_database: 
+    label: "Should the onset to death delay distribution be extracted directly from the epiparameter package?"
+    value: TRUE
+  onset_death_mean:
+    label: "Mean of the distribution for the time delay from disease onset to death if not using value from epiparameter. Ignored if `use_epiparameter_database = TRUE`."
+    value: 8
+  onset_death_sd: 
+    label: "Standard deviation for the time delay from disease onset to death if not using value from epiparameter. Ignored if `use_epiparameter_database = TRUE`."
+    value: 2
+  onset_death_dist: 
+    label: "Choice of probability distribution for the delay from onset to death if not using value from epiparameter. Ignored if `use_epiparameter_database = TRUE`."
+    value: "gamma"
+    choices: ["beta", "binom", "cauchy", "chisq", "exp", "f", "gamma", "geom", "hyper", "lnorm", "logis", "nbinom", "norm", "pois", "smirnov", "t", "tukey", "unif", "weibull", "wilcox"]
+  cfr_estimator: 
+    label: "Which method to use for CFR estimation"
+    value: "cfr"
+    choices: ["cfr", "manual", "EpiNow2"]
```

Comment:
Let's maybe just keep the parameters that have been implemented for now so we can go ahead and merge this first PR
```suggestion
params:
  data_file: 
    label: "Name of file containing cases and deaths over time"
    value: "data/Marburg_EqGuinea_linelist.csv"
    input: file
  disease_name: 
    label: "Name of the disease of interest, also used to select diseases in the epiparameter database if `use_parameter_database = TRUE`."
    value: "Ebola"
  data_columns:
    label: "Columns present in your dataset"
    value: "Dates of onset and total deaths only"
    choices: ["Dates of onset and total deaths only", "Dates of onset and death", "Dates of onset and whether case died"]
  use_epiparameter_database: 
    label: "Should the onset to death delay distribution be extracted directly from the epiparameter package?"
    value: TRUE
  onset_death_mean:
    label: "Mean of the distribution for the time delay from disease onset to death if not using value from epiparameter. Ignored if `use_epiparameter_database = TRUE`."
    value: 8
  onset_death_sd: 
    label: "Standard deviation for the time delay from disease onset to death if not using value from epiparameter. Ignored if `use_epiparameter_database = TRUE`."
    value: 2
  onset_death_dist: 
    label: "Choice of probability distribution for the delay from onset to death if not using value from epiparameter. Ignored if `use_epiparameter_database = TRUE`."
    value: "gamma"
    choices: ["beta", "binom", "cauchy", "chisq", "exp", "f", "gamma", "geom", "hyper", "lnorm", "logis", "nbinom", "norm", "pois", "smirnov", "t", "tukey", "unif", "weibull", "wilcox"]
```